src/components/Thanks/ThanksLayoutV2.vue

```diff
@@ -415,10 +415,16 @@ export default {
 			type: Boolean,
 			default: true
 		},
+		thanksSocialShareVersion: {
+			type: String,
+			default: ''
+		}
 	},
 	data() {
 		let visibleSection = 'receipt';
-		if (this.showGuestUpsell) {
+		if (this.showShare && this.thanksSocialShareVersion === 'b') {
+			visibleSection = 'share';
+		} else if (this.showGuestUpsell) {
```

Comment:
@emuvente Seems like the `showGuestUpsell` still needs to be the primary here.

Reply:
@edwinf-barreram We have clarification on this now. We should exclude guest checkout users from this experiment. So please change this part so that `this.showGuestUpsell` is checked first.

Reply:
done!